### PR TITLE
Improved docker compose file so can be run for ES6 and ES7

### PR DESCRIPTION
**Pull request by andrewgkew**
1. Updated compose file to include the SW_STORAGE env var otherwise it cant be overwritten when which between ES6 and ES7
2. Add missing env var SW_VERSION into env files
3. Updated versions of ES and SW to latest versions for both ES6 and ES7

This has been tested on docker Docker version 20.10.8, build 3967b7d on Ubuntu 18.04
And without the SW_STORAGE var added to compose file when running using the .env.es7 env file the OAP container fails, and without SW_VERSION the docker compose command does not work (you could pass it into the command but this seems cleaner)

### CODE REVIEW

**Comment by wu-sheng** [comment]
It is fine to upgrade ES server version for testing. But it seems to be not required.

**Comment by andrewgkew** [comment]
Agreed version upgrade is not required, but thought I would do it while I was making this change. Happy to remove that change from the functional change.

**Comment by wu-sheng** [comment]
We don't recommend you are using ElasticSearch SSPL licensed released. 
Take a lool at SkyWalking main repo storage doc, we shared with you this, and indicated the latest Apache 2.0 releases.

**Comment by wu-sheng** [comment]
If you want to upgrade, should be stopped at there. Elastic forces ASF community has to stop at there.

**Comment by andrewgkew** [comment]
Are you talking about pulling the images from https://www.docker.elastic.co/ 

i.e docker.elastic.co/elasticsearch/elasticsearch

The images are available on DockerHub using Elastic License 2.0

DIdnt realize about the SSPL licence will look into that.

I am happy to adjust the compose file to use dockerhub images instead if thats what you are saying?

File: compose/.env.es6 [hunk 1]
```diff
@@ -14,6 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ES_VERSION=6.8.8
+ES_VERSION=6.8.18
 SW_STORAGE=elasticsearch
 IMAGE_TAG=es6
+SW_VERSION=8.7.0
```
Comment (JaredTan95) [inline compose/.env.es6]: `SW_VERSION` set in https://github.com/apache/skywalking-docker/blob/master/Makefile#L58

**Comment by wu-sheng** [comment]
SSPL does not relate to docker, it affects the binary.
https://skywalking.apache.org/docs/main/v8.7.0/en/setup/backend/backend-storage/#elasticsearch

7.10 is the last Apache 2.0 licensed binary. I didn'tcheck ElasticSearch 6, but I think there is a red line.

**Comment by andrewgkew** [comment]
> Are you talking about pulling the images from https://www.docker.elastic.co/
> 
> i.e docker.elastic.co/elasticsearch/elasticsearch
> 
> The images are available on DockerHub using Elastic License 2.0
> 
> DIdnt realize about the SSPL licence will look into that.
> 
> I am happy to adjust the compose file to use dockerhub images instead if thats what you are saying?

Just seen in your docs that the licence is as of version 7.11

 This license change is effective from Elasticsearch version 7.11.

So can adjust the versions accordingly so that they dont clash with Apache licence

**Comment by andrewgkew** [comment]
> SSPL does not relate to docker, it affects the binary.
> https://skywalking.apache.org/docs/main/v8.7.0/en/setup/backend/backend-storage/#elasticsearch
> 
> 7.10 is the last Apache 2.0 licensed binary. I didn'tcheck ElasticSearch 6, but I think there is a red line.

Yup just seen that on your docs. Makes sense.

File: compose/.env.es6 (hunk 1 above)
Comment (andrewgkew) [reply compose/.env.es6]: I was using the docker compose files by themselves not using make. But that makes sense (no pun instead)

**Comment by wu-sheng** [comment]
We need a check on ElasticSearch 6. I think we can't use latest too.

File: compose/.env.es6 (hunk 1 above)
Comment (wu-sheng) [reply compose/.env.es6]: It is better to update doc about this env. We don't recommend hardcode version.

Comment (JaredTan95) [reply compose/.env.es6]: we use https://github.com/apache/skywalking-docker/blob/master/8/v8.7.0.sh  to export `SW_VERSION` when we dev and test. and I agree with @wu-sheng , do not harcode here.

Comment (andrewgkew) [reply compose/.env.es6]: Makes sense. Have pushed a change removing SW_VERSION

File: compose/.env.es6 [hunk 2]
```diff
@@ -14,6 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ES_VERSION=6.8.8
+ES_VERSION=6.8.18
```
Comment (wu-sheng) [inline compose/.env.es6:17]: Confirmed, https://github.com/elastic/elasticsearch/blob/v6.8.18/LICENSE.txt 6.8.18 is still Apache 2.0 License.

File: compose/.env.es7 [hunk 3]
```diff
@@ -14,6 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ES_VERSION=7.5.0
+ES_VERSION=7.10.0
```
Comment (wu-sheng) [inline compose/.env.es7:17]: https://github.com/elastic/elasticsearch/blob/v7.10.2/LICENSE.txt 7.10.2 is the last 7.x Apache 2.0 release.

**Review by wu-sheng — COMMENTED** [review]
Generally good to me, some nits in comments.

@JaredTan95 @kezhenxu94 @hanahmily Please recheck.

**Review by kezhenxu94 — APPROVED** [review]
LGTM

**Pull request merged**